uptrain/framework/remote.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, settings: Settings = None) -> None:
     def evaluate(
         self,
         data: list[dict],
-        checks: list[t.Union[Evals, ParametricEval]],
+        checks: list[t.Union[Evals, ParametricEval, dict]],
```

Comment:
Should take scenario_description as input?

Reply:
The scenario_description is already present in checks. It is populated in EvalLLM.